tools/theme/package.json

```diff
@@ -108,18 +108,50 @@
             "development": "./spectrum-two/scale-large-core-tokens.dev.js",
             "default": "./spectrum-two/scale-large-core-tokens.js"
         },
+        "./spectrum-two/scale-large.js": {
+            "development": "./spectrum-two/scale-large.dev.js",
+            "default": "./spectrum-two/scale-large.js"
+        },
         "./spectrum-two/scale-medium-core-tokens.js": {
             "development": "./spectrum-two/scale-medium-core-tokens.dev.js",
             "default": "./spectrum-two/scale-medium-core-tokens.js"
         },
+        "./spectrum-two/scale-medium.js": {
+            "development": "./spectrum-two/scale-medium.dev.js",
+            "default": "./spectrum-two/scale-medium.js"
+        },
         "./spectrum-two/theme-dark-core-tokens.js": {
             "development": "./spectrum-two/theme-dark-core-tokens.dev.js",
             "default": "./spectrum-two/theme-dark-core-tokens.js"
         },
+        "./spectrum-two/theme-dark.js": {
+            "development": "./spectrum-two/theme-dark.dev.js",
+            "default": "./spectrum-two/theme-dark.js"
+        },
+        "./spectrum-two/theme-darkest-core-tokens.js": {
+            "development": "./spectrum-two/theme-darkest-core-tokens.dev.js",
+            "default": "./spectrum-two/theme-darkest-core-tokens.js"
+        },
+        "./spectrum-two/theme-darkest.js": {
+            "development": "./spectrum-two/theme-darkest.dev.js",
+            "default": "./spectrum-two/theme-darkest.js"
+        },
         "./spectrum-two/theme-light-core-tokens.js": {
             "development": "./spectrum-two/theme-light-core-tokens.dev.js",
             "default": "./spectrum-two/theme-light-core-tokens.js"
         },
+        "./spectrum-two/theme-light.js": {
+            "development": "./spectrum-two/theme-light.dev.js",
+            "default": "./spectrum-two/theme-light.js"
+        },
+        "./spectrum-two/theme-lightest-core-tokens.js": {
```

Comment:
I was following the lead of the Express system, which also doesn't have `lightest` or `darkest` themes but nonetheless includes files for loading `lightest` and `darkest`. These files actually actually just load the `light` and `dark` values under the hood.

My assumption was that this was done to avoid failing ungracefully in the case where someone populates the `system` and `color` properties of  `<sp-theme>` with a non-existent pairing.

My instinct is that a more principled approach would be to rework `<sp-theme>` so that you simply get dev mode warnings when trying invalid combinations, rather than using this "facade" approach within each theme...but a) I'm not sure whether any customers of the Express theme might be relying on the status quo; and b) again, my general approach to this PR was to get the `spectrum-two` theme working with minimal changes to the code.

I am open to going a different way here, but my instinct remains to take a light touch for now, on the assumption that we'll likely end up making significant changes to the whole theming mechanism in the 1.x => 2.0 transition, and that would be a better time to revisit the approach here.

Reply:
Understood, and that does make sense. I agree with the "let's keep changes to a minimum" approach for now. It's definitely something we can address later on.

Reply:
If nothing else, this provides yet another good data point for us to look at, whenever we take an holistic approach to theming, "systems", sp-theme,...